app/uk/gov/hmrc/excisemovementcontrolsystemapi/connectors/MessageConnector.scala

```diff
@@ -69,16 +69,24 @@ class MessageConnector @Inject() (
       .setHeader("Authorization" -> s"Bearer $bearerToken")
       .execute[HttpResponse]
       .flatMap { response =>
-        if (response.status == OK) Future.fromTry {
-          for {
-            response <- parseJson[EISConsumptionResponse](response.body)
-            messages <- getMessages(response)
-            count    <- countOfMessagesAvailable(response.message)
-          } yield GetMessagesResponse(messages, count)
-        }
-        else {
-          logger.warn(s"[MessageConnector]: Invalid status returned: ${response.status}")
-          Future.failed(new RuntimeException("Invalid status returned"))
+        response.status match {
```

Comment:
Won't the client of this (MessageService) still fail, causing the Scheduled job to still fail? is all we've done here just give different error/warning messages for different statuses or am I missing something